### DISCUSSION TITLE
mohon pencerahanya mas rio, saya tidak bisa repairing hp ke qr

### DISCUSSION THREAD

**Discussion by bramtintonnugroho** [kirimwa-id/forum#163] [Tanya Jawab]
mohon pencerahanya mas rio, saya tidak bisa repairing hp ke qr

**Comment by rioastamal** [comment]
Pertanyaan tolong diperjelas apakah sebelumnya bisa pairing dan sekarang tidak bisa atau seperti apa.

**Reply by bramtintonnugroho** [reply]
jadi ketika saya akses usl ini, https://api.kirimwa.id/v1/qr?device_id=iphone-x-pro, saya dapat balikan image_url=> nah pas saya reapiring ke wa web pribadi saya tidak bisa

**Reply by bramtintonnugroho** [reply]
![image](https://user-images.githubusercontent.com/38525521/214198336-160271a7-961f-45bf-9af8-8c0489e62f3b.png)

**Reply by bramtintonnugroho** [reply]
sudah saya coba get QR lagi, errornya masih sama
